pom.xml

```diff
@@ -381,8 +381,8 @@
 
         <!-- Orbit versions-->
         <opensaml.version>3.3.1</opensaml.version>
-        <opensaml2.wso2.version>3.3.1.wso2v1</opensaml2.wso2.version>
-        <opensaml2.wso2.osgi.version.range>[3.3.1,3.4.0)</opensaml2.wso2.osgi.version.range>
```

Comment:
is opensaml2.wso2.osgi.version.range not used anywhere?

Reply:
Fixed